include/openssl/evp.h

```diff
@@ -63,6 +63,7 @@ struct evp_cipher_ctx_st {
     int encrypt;
     int iv_len;           // default: DEFAULT_IV_LEN.
     bool iv_set;          // boolean marks if iv has been set. Default:false.
+    int key_len;
```

Comment:
Go on, it will feel unloved if its the only line without a comment.

Reply:
Done.